third_party/catch.hpp

```diff
@@ -0,0 +1,14697 @@
+/*
```

Comment:
I think adding catch as a submodule would be better. If they update the library, we can integrate faster.